packages/pebble-shared/icons.json

```diff
@@ -77,6 +79,7 @@
   "home-filled",
   "home",
   "house-status",
+  "how_to_reg",
```

Comment:
instead of `_`, I think we should use `-` for consistency. Any way we will not be able to keep it same as material icons all the time, so let's keep it as close as possible.

Reply:
fixed in e3a2273